index.js

```diff
@@ -0,0 +1,46 @@
+require('dotenv').load();
+
+const http = require('http');
+const path = require('path');
+const express = require('express');
+var methods = require('./src/server.js');
+const tokenGenerator = methods.tokenGenerator;
+const makeCall = methods.makeCall;
+const placeCall = methods.placeCall;
+const incomingCall = methods.incomingCall;
+var twilio = require('twilio');
+
+// Create Express webapp
+const app = express();
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.get('/accessToken', function(request, response) {
+  const identity = request.query.identity || 'identity';
+  response.send(tokenGenerator(identity));
+});
+
+app.get('/makeCall', function(request, response) {
+  const to = request.query.to;
+  response.setHeader("Content-Type", "text/xml");
```

Comment:
Is `text/xml` the preferred mime type for this? Seems a bit odd to use this type.